setup.cfg

```diff
@@ -58,6 +58,7 @@ install_requires =
     certifi>=2017.4.17
     typing_extensions>=4.3,<5
     filelock>=3.5,<4
+    sortedcontainers
```

Comment:
Adding dependency for using `SortedSets` for `QueryContextCache`. I have added this requirement in `setup.cfg` and `requirements_3[7,8,9,10].reqs`. Is there any other place I need to update this?